tests/resources/tliba/src/tliba/random/rng.py

```diff
@@ -14,7 +14,4 @@ def draw_normal_samples():
 
 def draw_bernoulli_samples():
     """Create table consisting of 5000 draws from a bernoulli distribution where the probability is drawn form a Beta(2, 5)."""
-    return pd.DataFrame(
-        np.random.binomial(1, draw_beta_samples().values.flatten(), 5000),
-        columns=["binomial"],
-    )
+    return pd.DataFrame(np.random.binomial(1, draw_beta_samples().values.flatten(), 5000), columns=["binomial"])
```

Comment:
Can we avoid any style changes in the tests directory? The module hash tests are failing because of this change

Reply:
Removed all style updates in test/